### PR TITLE
Update to 10 minute tutorial

### LINKED ISSUE #367: 10-Minute tutorial uses different versions of Cucumber
Looks like the 10 Minute tutorial is using:

`   "-DarchetypeVersion=4.2.6.1"               \`  for maven, `def cucumberVersion = '4.3.1'` for gradle, and imports from: 4.7:

```
import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;
```

It'd be best to pin the tutorial against 4.2.6.

https://cucumber.io/docs/guides/10-minute-tutorial/

### PR DESCRIPTION
**Pull request by anna-engle**
Removed the snippet for adding the `RunCucumberTest.java` file completely, as version 4.2.6 automatically creates this file for you when you generate the archetype.

Fixes: #367

### CODE REVIEW

**Comment by mpkorstanje** [comment]
I think that part of code was part of the gradle tutorial.

Maybe we should remove the Gralde tutorial parts. The interleaving between Maven and Gradle makes this really messy and hard to follow.

@mlvandijk wdyt?

**Comment by mpkorstanje** [comment]
And I've made a `4.8.0.0`  version of the archetype. Might be useful.

**Comment by anna-engle** [comment]
@mpkorstanje I updated the archetype version to 4.8.0.0 and the Gradle section to include the command to convert from a Maven project.  Instead of using the template provided in the tutorial, we can add the configs/dependencies/task necessary to build.gradle.

File: content/docs/guides/10-minute-tutorial.md [hunk 1]
```diff
@@ -125,57 +125,29 @@ Open the project in IntelliJ IDEA:
...
+        javaexec {
+            main = "io.cucumber.core.cli.Main"
+            classpath = configurations.cucumberRuntime + sourceSets.main.output + sourceSets.test.output
+            args = ['--plugin', 'pretty', '--glue', 'gradle.cucumber', 'src/test/resources']
+        }
+    }
 }
 ```
+Note that you also need to add the necessary dependencies/configurations to `build.gradle` depending on which version of Gradle you are using.
+See the [Build Tools](https://cucumber.io/docs/tools/java/#gradle) section.
```
Comment (mpkorstanje) [inline content/docs/guides/10-minute-tutorial.md]: It's not done consistently throughout the documentation but let's use a relative URL here. They're shorter, more readable and guaranteed to stay in the same domain.

```suggestion
See the [Build Tools](/docs/tools/java/#gradle) section.
```

**Comment by anna-engle** [comment]
@mpkorstanje I agree with what you're saying on it being hard to follow, especially now that this tutorial includes js/kotlin/ruby implementations.  Maybe we could implement something like the following: https://www.cockroachlabs.com/docs/stable/training/cluster-startup-and-scaling.html (see the Mac and Linux buttons)

File: content/docs/guides/10-minute-tutorial.md [hunk 2]
```diff
@@ -125,57 +125,29 @@ Open the project in IntelliJ IDEA:
 
 **With Gradle**
 
-You can use the following `build.gradle` file as an example template. Use your IDE
-to start a new Gradle-based project.
+The easiest way to create this sample Cucumber project using Gradle is to convert the above generated Maven archetype into a Gradle project.
```
Comment (mpkorstanje) [inline content/docs/guides/10-minute-tutorial.md]: With #389 in mind I think we should rephrase:

```suggestion
One way to create this sample Cucumber project using Gradle is to convert the above generated Maven archetype into a Gradle project.
```

**Review by mpkorstanje — APPROVED** [review]
Awesome! This looks much better already. Cheers. If nobody else has any comments I'll merge this in a few days.

I've added a few suggested edits. I think Github should let you commit them automatically but I'm not sure (I've not used this feature much yet).

> Maybe we could implement something like the following: https://www.cockroachlabs.com/docs/stable/training/cluster-startup-and-scaling.html (see the Mac and Linux buttons)

That would be nice. I'm not sure how easy it would be too add. I do know that @mlvandijk and @mattwynne have been fighting with Hugo to make the current language thing possible.

Perhaps we can add Gralde and Maven as language options?

File: content/docs/guides/10-minute-tutorial.md (hunk 2 above)
Comment (annabaker) [reply content/docs/guides/10-minute-tutorial.md]: Making language more inclusive

**Comment by aslakhellesoy** [comment]
Hi @annabaker,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mpkorstanje** [comment]
Cheers!

**Comment by mlvandijk** [comment]
@mpkorstanje @annabaker I'd love to have options for Maven/Gradle (and possibly IDE) in the docs, but am not sure how to do so, especially in combination with the current language sections.
I like the way Spring does this, for example: https://spring.io/guides/gs/rest-service/
